src/appleseed/renderer/modeling/bsdf/glossybrdf.cpp

```diff
@@ -368,10 +368,7 @@ namespace
                 }
                 break;
 
-              default:
-                assert(false);
-                pdf = 0.0f;
-                break;
+              assert_otherwise;
```

Comment:
I'm not sure this change and other similar changes in this PR are a good idea, IIRC some compilers (gcc 7?) will complain that `pdf` is potentially uninitialized.